dap/BESDapResponseBuilder.cc

```diff
@@ -80,11 +83,13 @@
 #include <mime_util.h>	// for last_modified_time() and rfc_822_date()
 #include <escaping.h>
 #include <util.h>
+#if USE_LOCAL_TIMEOUT_SCHEME
```

Comment:
On January 4, 2016 at 5:45:50 PM, Patrick West (notifications@github.com) wrote:

In dap/BESDapResponseBuilder.cc:

> @@ -80,11 +83,13 @@
>  #include <mime_util.h>      // for last_modified_time() and rfc_822_date()
>  #include <escaping.h>
>  #include <util.h>
> +#if USE_LOCAL_TIMEOUT_SCHEME
Are you using a macro here because the alarm doesn't work on some systems?

No. I had to competing timeout schemes in the code (here and in libdap::DDS) and I’ve replaced both with the new code. But, I’m hedging my bets by keeping the old code in the sources but excluding it from compilation using this #if. 

Thanks for looking at this. NB: I just wrote another test and pushed it (in module/debug_functions)

James


—
Reply to this email directly or view it on GitHub.


-- 
James Gallagher
jgallagher@opendap.org